resources/services/s3/buckets.go

```diff
@@ -787,3 +799,41 @@ func resolveBucketTagging(ctx context.Context, meta schema.ClientMeta, resource
 	}
 	return resource.Set("tags", tags)
 }
+
+func resolveBucketOwnershipControls(ctx context.Context, meta schema.ClientMeta, resource *schema.Resource, bucketName, bucketRegion string) error {
+	c := meta.(*client.Client)
+	svc := c.Services().S3
+
+	getBucketOwnershipControlOutput, err := svc.GetBucketOwnershipControls(ctx, &s3.GetBucketOwnershipControlsInput{Bucket: aws.String(bucketName)}, func(options *s3.Options) {
+		options.Region = bucketRegion
+	})
```

Comment:
Because the list S3 bucket for every region returns the same thing, does this mean we will grab ALL s3 bucket details for every region that the user specifies in the `config.hcl`?

Reply:
Not actually an issue